client/src/components/BreadCrumbs/BreadCrumbItem.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Link } from 'react-router-dom';
 import { themeColor } from '@amsterdam/asc-ui';
+import { UnderlinedLink } from '@components/LocalAsc/LocalAsc';
```

Comment:
Hier ruil je Link uit react-router-dom in voor een gewone li toch? Weet niet of dat de bedoeling is.

Reply:
Hiermee gebruiken we het Link component uit de Amsterdam Styled Components library, alleen underlinen we deze wel by default.